src/napari/utils/logo.py

```diff
@@ -0,0 +1,56 @@
+import re
+from datetime import date
+from importlib import resources
+from pathlib import Path
+
+
+def available_logos():
+    logo_dir = Path(resources.files('napari').joinpath('resources', 'logos'))
+    variants = ['auto']
+    for logo in Path(logo_dir).glob('*.svg'):
```

Comment:
```suggestion
    for logo in Path(logo_dir).glob('*plain-light.svg'):
```
Then not need to de match on path

Reply:
Yes we do, because there are are variants in there (the dark mode ones as well). I think we should always go for the light/dark version depending on theme.

Reply:
I do not understand. Just a line below you drop the dark version...

Reply:
I don't follow: one line below there's no reference to dark versions?

This function lists how many "variants" exist (gradient, flat, christmas, etc), it does not also give the two themed versions.

Reply:
Your code is:
```python
for logo in Path(logo_dir).glob('*.svg'):
	if match := re.match(r'(.*)-plain-light', logo.stem):
        variants.append(match.group(1))
```
The equivalent is 
```python
for logo in Path(logo_dir).glob('*-plain-light.svg'):
    variants.append(logo.stem.rsplit('-', 2))[0]
```

Reply:
AAAh! Lol, ok, somehow I completely misunderstood the github suggestion and read it backwards. Ignore what I said xD